### PR TITLE
Added django ORM support

### PR DESCRIPTION
**Pull request by lingster**
Add pagination if you are using django as the orm with fastapi.

### CODE REVIEW

**Comment by uriyyo** [comment]
@lingster Could you please add tests to cover this feature?

File: fastapi_pagination/ext/django.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+from pydantic import BaseModel
+
+
+# To convert from a django ORM model to a pydantic Model, provide a BaseModel with from_django() method to map fields
+# from django to pydantic
```
Comment (uriyyo) [inline fastapi_pagination/ext/django.py:8]: I am curious about, why we can't use pydantic `orm_mode =True` to parse models?

**Review by uriyyo — CHANGES_REQUESTED** [review]

**Comment by uriyyo** [comment]
Hi @lingster,

Could you please rebase on master? I will merge it and add tests by myself)

**Comment by uriyyo** [comment]
@lingster Thanks)

**Comment by lingster** [comment]
Hi @urlyyo,
That should be done.

**Pull request merged**